maas/resource_maas_vlan_dhcp_test.go

```diff
@@ -0,0 +1,491 @@
+package maas_test
+
+import (
+	"fmt"
+	"regexp"
+	"strconv"
+	"strings"
+	"terraform-provider-maas/maas"
+	"terraform-provider-maas/maas/testutils"
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/acctest"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/terraform"
+)
+
+func TestAccMAASVLANDHCP_basic(t *testing.T) {
+	// Test variables
+	fabricName := acctest.RandomWithPrefix("tf-basic")
+	cidr := testutils.GenerateRandomCIDR()
+	networkPrefix := testutils.GetNetworkPrefixFromCIDR(cidr)
+	startIP, endIP := networkPrefix+".2", networkPrefix+".5"
+	startIP2, endIP2 := networkPrefix+".6", networkPrefix+".10"
+	rackController := "maas-dev"
```

Comment:
this one should be given as input with an environment variable. This environment variable should be checked that it is set in order to run the test.